components/brave_wallet/browser/eth_tx_controller.cc

```diff
@@ -171,31 +171,21 @@ void EthTxController::AddUnapprovedTransaction(
 
   auto tx_ptr = std::make_unique<EthTransaction>(*tx);
 
```

Comment:
nit: please remove this extra new line here between tx_ptr and gas_limit.

Reply:
Done